README.md

```diff
@@ -136,4 +136,28 @@ b). Implement an endpoint to retrieve task comments and display them in the task
 
 1. Create a new table to store task comments. The table could be named task_comment and could have the following columns: id (primary key), task_id (foreign key to reference the task the comment belongs to), comment (the actual comment text), and created_at (timestamp for when the comment was added).
 2. Add a new API endpoint that allows users to add comments to a task. This endpoint should receive the task ID and the comment text as input, and then insert the comment into the task_comment table.
-3. Add a new API endpoint that allows users to retrieve all comments for a specific task. This endpoint should receive the task ID as input and return a list of comments associated with that task.
\ No newline at end of file
+3. Add a new API endpoint that allows users to retrieve all comments for a specific task. This endpoint should receive the task ID as input and return a list of comments associated with that task.
+
+## Unit tests:
+
+1. Add unit tests to all your code you have created so far.
+2. Aim to achieve 80% code coverage.
+
+## Integration tests:
+
+1. Add integration tests to test your API functionality.
+
```

Comment:
we could also get them to generated a test report
https://github.com/vakenbolt/go-test-report